RevenueCatUI/CustomerCenter/Actions/CustomerCenterView+Actions.swift

```diff
@@ -172,6 +174,19 @@ extension CustomerCenterView {
                 }
         }
     }
+
+    struct OnChangePlansSelected: ViewModifier {
+        let handler: ChangePlansHandler
+
+        func body(content: Content) -> some View {
+            content
+                .onPreferenceChange(ChangePlansSelectedPreferenceKey.self) { wrapperSubscriptionGroupID in
+                    if let subscriptionGroupID = wrapperSubscriptionGroupID?.value {
```

Comment:
I am not sure, but with the current code, I believe this can be an empty `""` string? https://github.com/RevenueCat/purchases-ios/pull/5379/files#diff-8182f46c9c24cf0ef51c67a65739033ca5c8205ac5498ebb192e6ed81cddc82fR185